app/models/user.rb

```diff
@@ -2,4 +2,8 @@ class User < ApplicationRecord
   has_many :posts, foreign_key: :author_id
   has_many :likes
   has_many :comments
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(5)
```

Comment:
- [ ] The project reqires that you  display the 3 most recent posts for a given user not 5. Kindly update  it 👍🏽 .